Source/HelixToolkit.Wpf/Controls/HelixViewport3D.cs

```diff
@@ -3224,6 +3224,7 @@ public override void OnApplyTemplate()
                 if (this.adornerLayer != null)
                 {
                     this.adornerLayer.Child = this.viewport;
+                    adornerLayer.IsHitTestVisible = false;

```

Comment:
no side effects from this change?
I guess the adorner layer never needs hit testing.
Maybe it should be set in xaml instead: https://github.com/helix-toolkit/helix-toolkit/blob/master/Source/HelixToolkit.Wpf/Themes/Generic.xaml#L94